docs/components/custom_components.md

```diff
@@ -0,0 +1,115 @@
+
+```diff
+   ) : isError ? (
+     <div>Unable to load Discovery projects. Please check your console for more details.</div>
+   ) : (
+-    <DiscoverySearch searchClient={searchClient} projectId={projectId}>
++    <DiscoverySearch
++      searchClient={searchClient}
++      projectId={projectId}
++      overrideQueryParameters={{
++        passages: {
++          enabled: true,
++          max_per_document: 3,
++          characters: 850,
++          fields: ['title', 'text'],
++          find_answers: true,
++          max_answers_per_document: 1
++        },
++        _return: ['document_passages', 'extracted_metadata']
++      }}
++    >
+       <AppView />
+     </DiscoverySearch>
+   );
+```
+
+The overall details are explained in the article, but I did want to point out a couple things. First, `fields` should be set to reflect the data in your project/collection. This property tells Discovery in which document fields to look for answers. Second, we're setting `_return` to a set of top-level fields in the document results. Only the fields listed will be returned, cutting down on the response size (especially since we're not listing `text` and `html`, which can be quite large).
+
+(Note: Besides `overrideQueryParameters`, you can also update the query parameters by calling the `setSearchParameters` function on the `SearchApi` context.)
+
+Now let's build our custom component. The Discovery Components use [React Context](https://reactjs.org/docs/context.html) to pass data and give access to their API. The results of a search are provided in the `searchResponseStore` property of `SearchContext`. From within that, we'll find the most relevant `answers`, based on their `confidence` score.
+
+Copy the following code into `src/App.js`:
+
+```javascript
+// Show top 3 answers from within search results
+function Answers() {
+  // Use the context provided by <DiscoverySearch> to get search results
+  const {
+    searchResponseStore: { data: searchResponse }
+  } = useContext(SearchContext);
+
+  if (!searchResponse) {
+    return null;
+  }
+
+  const top3Answers = searchResponse.results
+    // extract document passages from documents in the results...
+    .reduce((acc, doc) => acc.concat(doc.document_passages), [])
+    // extract answers from the passages...
+    .reduce((acc, psg) => acc.concat(psg.answers), [])
+    // sort based on confidence score...
+    .sort((a, b) => b.confidence - a.confidence)
+    // and take the top 3
+    .slice(0, 3);
+
+  // render the answers as an ordered list
+  return (
+    <div>
+      <ol>
+        {top3Answers.map(answer => (
+          <li>{answer.answer_text}</li>
+        ))}
+      </ol>
+    </div>
+  );
+}
+```
+
+(Note: no styling provided for this component.)
```

Comment:
should we say anything about what file they can use to add style info for this? 
Note: No style is defined for the Answers class (is it a class?), but you can add a style definition for it in the x file.

Reply:
The way I see it, this is just a quick tutorial and I didn't want to get bogged down in unneeded details (like styling). Perhaps I should just remove this note entirely.